jenkins_pipelines/environments/common/pipeline-pull-request.groovy

```diff
@@ -15,6 +15,8 @@ def run(params) {
         terraform_init = true
         rake_namespace = 'cucumber'
         env_number = 6
+        repo_dir = '/home/jenkins/jenkins-build/workspace/'
+        this_host = env.BUILD_URL.split('/')[2].split(':')[0]
```

Comment:
Rename `this_host` by `fqdn_jenkins_node`

Reply:
The splits are not super elegant, but probably other options will require especial approval from the Jenkins Master, so it's fine.